docs/structure.md

```diff
@@ -4,7 +4,7 @@ This document describes the CsWinRT repository organization. Documentation and s
 
 ## [`nuget`](../nuget)
 
-Contains source files for producing a C#/WinRT NuGet package, which is regularly built, signed, and published to nuget.org by Microsoft.  The C#/WinRT NuGet package contains the cswinrt.exe compiler, and both versions of the winrt.runtime.dll.
+Contains source files for producing a C#/WinRT NuGet package, which is regularly built, signed, and published to nuget.org by Microsoft.  The C#/WinRT NuGet package contains the **cswinrt.exe** compiler and the runtime assembly, **WinRT.Runtime.dll**.
```

Comment:
the formatting here in structure.md is different from in readme.md (e.g., **winrt.runtime.dll** vs `winrt.runtime.dll`)